back/spec/acceptance/admin_publications_spec.rb

```diff
@@ -279,4 +279,33 @@
       end
     end
   end
+
+  context 'not logged in' do
+    before do
+      @projects = ['published','archived','draft','published','archived']
+        .map { |ps|  create(:project, admin_publication_attributes: {publication_status: ps})}
+
+      next unless CitizenLab.ee?
+
+      @folder = create(:project_folder, projects: @projects.take(2))
+      @empty_draft_folder = create(:project_folder, admin_publication_attributes: {publication_status: 'draft'})
+    end
+
+    get "web_api/v1/admin_publications/status_counts" do
+      example "Get publication_status counts for top-level admin publications" do
```

Comment:
It would be cleaner if we could link somehow this example to the existing `get "web_api/v1/admin_publications/status_counts"` section.

E.g., we could move the same code in shared example https://relishapp.com/rspec/rspec-core/docs/example-groups/shared-examples

The second advantage (apart from linking) would be no code duplication.

Reply:
OK, but the expectations are different. For the admin we expect draft publication count, but for no-admin we don't. How could I share this example _and_ include that difference?

Reply:
Because this is blocking an epic iteration, I will merge this now, and look into reducing code duplication with a shared example later this week.